### ISSUE TITLE
Show intros with \SolutionsOnly

### ISSUE THREAD

**Issue by thorstengrothe** [Strauman/exercisebank#94]
I'm very busy with exams  and exercises at the moment, of course with exercisebank :-) I often use env
```
\begin{intro}
some information stuff for some following problems
\end{intro}
```
to declare some information for my students. This can be a graphics file or a tabular etc. The following problems refer to the information shown in intro. So I think it would be useful if `\SolutionsOnly` would show the corresponding intro environment as well. What do you think?

**Comment by Strauman** [comment]
Yes! I would maybe suggest some optional arguments sent to `\SolutionsOnly`?

**Comment by thorstengrothe** [comment]
Yes, good idea. For now I put a 

\nextproblem{tags{mytag}}

before every intro, this works too, but it is a bit inconvenient :-)
